cli/auth.go

```diff
@@ -207,6 +207,55 @@ func (c *viamClient) whoAmIAction(cCtx *cli.Context) error {
 	return nil
 }
 
+// OrganizationAPIKeyCreateAction corresponds to `organization api-key create`.
+func OrganizationAPIKeyCreateAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	if c.conf.Auth == nil {
+		return errors.New("not logged in: run the following command to login:\n\tviam login")
+	}
+	orgID := c.c.String(apiKeyCreateFlagOrgID)
+	keyName := c.c.String(apiKeyCreateFlagName)
+	if keyName == "" {
+		// Formats name as myusername@gmail.com-2009-11-10T23:00:00Z
+		keyName = fmt.Sprintf("%s-%s", c.conf.Auth.User.Email, time.Now().Format(time.RFC3339))
+		infof(cCtx.App.Writer, "using default key name of %q", keyName)
+	}
+	resp, err := c.createOrganizationAPIKey(orgID, keyName)
+	if err != nil {
+		return err
+	}
+	infof(cCtx.App.Writer, "successfully created key:")
+	fmt.Fprintf(cCtx.App.Writer, "key id: %s\n", resp.GetId())
+	fmt.Fprintf(cCtx.App.Writer, "key value: %s\n\n", resp.GetKey())
+	infof(cCtx.App.Writer, "keep this key somewhere safe; it has full write access to your organization")
```

Comment:
[nit] warning maybe?

Reply:
At first I thought warning was a bit too much but I guess you're right.